app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -384,19 +387,27 @@ const SelectOptions = ({
   </>
 );
 
-export const filterTargetingConfigs = (
+export const filterAndSortTargetingConfigs = (
   targetingConfigs: getConfig_nimbusConfig["targetingConfigs"],
   application: getExperiment_experimentBySlug["application"],
 ) =>
   targetingConfigs == null
     ? []
-    : targetingConfigs.filter(
-        (
-          targetingConfig,
-        ): targetingConfig is getConfig_nimbusConfig_targetingConfigs =>
-          targetingConfig !== null &&
-          Array.isArray(targetingConfig.applicationValues) &&
-          targetingConfig.applicationValues.includes(application),
-      );
+    : targetingConfigs
+        .filter(
+          (
+            targetingConfig,
+          ): targetingConfig is getConfig_nimbusConfig_targetingConfigs =>
+            targetingConfig !== null &&
+            Array.isArray(targetingConfig.applicationValues) &&
+            targetingConfig.applicationValues.includes(application),
+        )
+        .sort(
+          // sort with default value '' first, then alphabetical
+          (a, b): number =>
+            Number(b?.value === "") - Number(a?.value === "") ||
+            a?.label?.localeCompare(b?.label || "") ||
+            0,
```

Comment:
Yeah too bad we have to call out the special case for the empty value to be first, another way to do it would be to pull the empty value out, sort the remaining values, then insert the empty value back in, but I can't imagine that's any simpler or less special casey than this so good job 👍